1st draft for table10, 11, 12 (#102)
pharmaverse/falcon: quarto/table-templates/template-table_10.qmd

---
title: FDA Table 10
subtitle: Patients With Serious Adverse Events by System Organ Class and FDA Medical Query (Narrow), Safety Population, Pooled Analyses
format: html
---

::: panel-tabset
## Spec. Screenshot

![](../assets/images/screenshots/table_10.png){fig-align="center"}

## Table

```{r tbl, message=FALSE, warning=FALSE}
# Load Libraries & Data
library(dplyr)
library(falcon)

adsl <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "adsl")
adae <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "adae")

> # Pre-Processing - Ensure required variables FMQ01SC and FMQ01NAM exist in adae

Reviewer: ```suggestion
# Pre-Processing - Ensure required variables fmqsc_var and fmqnam_var exist in adae
```

| `show_colcounts` | (`flag`) Whether column counts should be printed.                                                                                                                                                                                                      | `TRUE`        |
| `arm_var`        | (`character`) Arm variable used to split table into columns.                                                                                                                                                                                           | `"ARM"`       |
| `fmqsc_var`      | (`character`) FMQ scope variable to use in table.                                                                                                                                                                                                      | `"FMQ01SC"`   |
| `fmqnam_var`     | (`character`) FMQ reference name variable to use in table.                                                                                                                                                                                             | `"FMQ01NAM"`  |

> | `fmq_scope`      | (`character`) FMQ scope, must be "NARROW" or "BROAD".                                                                                                                                                                                                  | `"NARROW"`    |

Reviewer: ```suggestion
| `fmq_scope`      | (`character`) FMQ scope, can be `"NARROW"` or `"BROAD"`.                                                                                                                                                                                                  | `"NARROW"`    |
```

| `lbl_overall`    | (`character`) If specified, an overall column will be added to the table with the given value as the column label.                                                                                                                                     | `NULL`        |
| `prune_0`        | (`flag`) Whether all-zero rows should be removed from the table.                                                                                                                                                                                       | `FALSE`       |

> | `na_level`       | (`character`) string to represent missing values.                                                                                                                                                                                                      | `"<Missing>"` |

Reviewer: ```suggestion
| `na_level`       | (`character`) String to represent missing values.                                                                                                                                                                                                      | `"<Missing>"` |
```
